src/main/app/case/definition.ts

```diff
@@ -2183,6 +2183,8 @@ export const enum Gender {
   MALE = 'Male',
   FEMALE = 'Female',
   OTHER = 'Other',
+  UNSPECIFIED = 'Unspecified',
```

Comment:
@vivek-sekhar Can we please remove UNSPECIFIED & use OTHER to make it consistent with pvt law & solicitor journey.